### ISSUE TITLE
Incorrect parsing of R CITATION files 

### ISSUE THREAD

**Issue by cboettig** [Impactstory/citeas-api#40]
http://citeas.org/cite/https://github.com/tidyverse/ggplot2 correctly finds the R CITATION file (yay!!) https://raw.githubusercontent.com/tidyverse/ggplot2/master/inst/CITATION, but fails to parse this citation correctly (misses most fields, and assumes it is bibtex article format instead of the declared bibtex format)

**Comment by jameshowison** [comment]
Thanks, that seems a relatively easy fix, ensureing that bibtex is parsed respecting the types (not assuming @article.  Although shouldn't that bibtex start with @book?  Is the formatting done in this CITATION file defined anywhere? I'm talking about citHeader and citEntry?

**Comment by cboettig** [comment]
right, exactly, should be a book.  R's `citEntry()` function is based upon and maps 1:1 with bibtex (note the `entry = "book"` in this example).   R will render these into bibtex too.

**Comment by caseydm** [comment]
Hello. I just made an update to improving parsing of R citation files. The bibtex download now accurately reflects the declared type in the citation. From what I can tell all fields with data in the citation are now reflected in the software. 

When the type is 'book' I removed the journal, volume, and number fields from the bibtex download but kept the pages field. Let me know if this is desired behavior.
